### PR TITLE
added FollowRedirects and AllowInsecure to LoadBalancerMonitor

### PR DESCRIPTION
**Pull request by FnuGk**
Missing fields in LoadBalancerMonitor
see [api doc](https://api.cloudflare.com/#load-balancer-monitors-properties)

### CODE REVIEW

File: load_balancing.go [hunk 1]
```diff
@@ -48,6 +48,9 @@ type LoadBalancerMonitor struct {
 	Interval      int                 `json:"interval"`
 	ExpectedBody  string              `json:"expected_body"`
 	ExpectedCodes string              `json:"expected_codes"`
+
+	FollowRedirects bool `json:"follow_redirects"`
```
Comment (jacobbednarz) [inline load_balancing.go]: Any reason why we aren't grouping these with the remainder of the fields?

**Comment by jacobbednarz** [comment]
@FnuGk Do you have any time to wrap this one up? Looks like the Terraform provider wants this functionality and it would be good to finish this one as you're quite close to getting this over the line.

File: load_balancing.go (hunk 1 above)
Comment (FnuGk) [reply load_balancing.go]: Nope, i fixed it

**Comment by jacobbednarz** [comment]
cc @patryk - this is ready for review 🍰

**Comment by patryk** [comment]
Many thanks, @FnuGk!

**Pull request merged**